app/src/main/res/values/colors.xml

```diff
@@ -7,4 +7,11 @@
     <color name="teal_700">#FF018786</color>
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
+
+    <!--    my colors-->
+    <color name="my_white">#ececec</color>
```

Comment:
it is now white, I guess it is some kind of light gray. right?

Reply:
It looks like white within that screen. Really close. And there is no real white color(#FFFFFF) in the whole mockup. At least not yet. And i know that it is not true white so I called it MY white. 

All colors names for the mockup start with "my..." so I guess it would be easier to find it while creating layouts